examples/ecs/README.md

```diff
@@ -1,6 +1,8 @@
 # Dojo ECS Example
 
-This repo contains a simple example of using the Dojo ECS system. It defines a simple game where a player has a limited set of moves and can move their position through executing the move system.
+This repo contains a simple example of using the Dojo ECS system. It defines a
+simple game where a player has a limited set of moves and can move their
+position through executing the move system.
```

Comment:
this was auto formatted by vscode, lmk if this is a problem i will remove formatting related stuff from this PR.

Reply:
done